modules/cnf-configuring-log-reduction-for-linuxptp.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-configuring-log-reduction-for-linuxptp_{context}"]
+= Configuring PTP log reduction
+
+The `linuxptp` daemon generates logs that you can use for debugging purposes. In telco or other deployment types that feature a limited storage capacity, these logs can add to the storage demand. Currently, the default logging rate is high, causing logs to rotate out in under 24 hours, which makes it difficult to track changes and identify problems.
```

Comment:
It feels like that there's intention to change this in the future. I would rephrase it in a way that if they're high, these are the issues you're facing:
```suggestion
The `linuxptp` daemon generates logs that you can use for debugging purposes. In telco or other deployment types that feature a limited storage capacity, these logs can add to the storage demand. If the default logging rate is high, the logs rotate out in under 24 hours, which makes it difficult to track changes and identify problems.
```

Then, you can go into explaining a solution, log reduction.

Reply:
I think suggestion changes the understanding so keeping with first way.